Updated hackathon starter kit (#9)
amadeus4dev/hackathon-starter: README.md

- [Python](https://github.com/amadeus4dev/amadeus-python)
- [Ruby](https://github.com/amadeus4dev/amadeus-ruby)
- [Node](https://github.com/amadeus4dev/amadeus-node)

> ## Cheatsheet

Reviewer: ```suggestion
## Cheat sheet
```

We normally bring some printed copies to the hackathons, but for your convenience check out the [web version](https://possible-quilt-2ff.notion.site/Cheat-sheet-e059caf4fcd342b78705f9f3d6f88f1d). 

Let's save the planet!

> ## Tutorials & Guides

Reviewer: ```suggestion
## Tutorials & guides
```

> In the [developer guides](https://amadeus4dev.github.io/developer-guides/examples/prototypes/) you can find tutorials to help you make the most of our APIs. 

Reviewer: ```suggestion
In the [Developer Guides](https://amadeus4dev.github.io/developer-guides/examples/prototypes/) you can find tutorials to help you make the most of our APIs. 
```

You can also find a list of [blog articles](https://developers.amadeus.com/blog/category/tutorials?page=1&count=5) and [videos](https://www.youtube.com/channel/UCwI48vMwtiE-hra2RAtk1PQ) to help you build travel apps with several Amadeus APIs and technologies.

## Postman

> If you love [Postman](https://www.getpostman.com/) as we do, you'll be glad to see [our public postman workspace](https://www.postman.com/amadeus4dev/workspace/amadeus-for-developers-s-public-workspace/collection/2672636-27471449-d2ca-a8c4-1399-6b0cfbddd079).

Reviewer: ```suggestion
If you love [Postman](https://www.getpostman.com/) as we do, you'll be glad to see [our public Postman workspace](https://www.postman.com/amadeus4dev/workspace/amadeus-for-developers-s-public-workspace/collection/2672636-27471449-d2ca-a8c4-1399-6b0cfbddd079).
```

> New to Postman? Don't miss our [How to play with Self-Service APIs with no code](https://developers.amadeus.com/blog/how-to-play-with-self-service-apis-with-no-code-using-postman) article in our blog.

Reviewer: ```suggestion
New to Postman? Don't miss the [How to play with Self-Service APIs with no code](https://developers.amadeus.com/blog/how-to-play-with-self-service-apis-with-no-code-using-postman) article in our blog.
```

## Examples

Explore all the self-contained [code examples](https://amadeus4dev.github.io/developer-guides/examples/code-example/) as well as [prototypes](https://amadeus4dev.github.io/developer-guides/examples/prototypes/). 

> ## Data-Collection

Reviewer: ```suggestion
## Data collection
```

> During the hackathon you'll use the `Test environment`, which means that our APIs are using mostly cache data. You can find which data is available on the [data collection](http://github.com/amadeus4dev/data-collection) repository.

Reviewer: ```suggestion
During the hackathon you'll use the `Test environment`, which means that our APIs are using cached data. You can find which data is available on the [data collection](http://github.com/amadeus4dev/data-collection) repository.
```
